### ISSUE TITLE
Plugin is not working with Vue 3

### ISSUE THREAD

**Issue by sadashivm** [Semantic-UI-Vue/Semantic-UI-Vue#489]
Hello Team,
Currently I'm using Vue 3 latest version. I tried to install this semantic-ui-vue in my project, package was installed but I'm getting some error.

1. Package installed,
![image](https://user-images.githubusercontent.com/10355473/115123366-53a5e500-9fda-11eb-9afd-9c09d8c98727.png)

2. Imported in main.js File,
![image](https://user-images.githubusercontent.com/10355473/115123427-9e276180-9fda-11eb-9da8-de7ce097a9a5.png)

3. Added simple button App.vue file,
![image](https://user-images.githubusercontent.com/10355473/115123438-b13a3180-9fda-11eb-9d4f-056439c86138.png)

4.  Its throwing an below error,
![image](https://user-images.githubusercontent.com/10355473/115123460-c4e59800-9fda-11eb-8139-cf5ee0043bb5.png)

Just wanted to know, is this plugin support vue 3 version?
I have see many framework like bootstrap, material ui but this semantic is awesome. No one will beat this framework.

Request to you please replay to above my question.

**Comment by bszyman** [comment]
Yes, I have had the same issue when I attempted to use Semantic-UI-Vue with VueJS 3. This is highly unfortunate because this really is a great UI framework to work with, plus it just looks great. The only other framework that meets my aesthetic sensibilities is Cappuccino, which oddly seems to be a more active project.

What can we do to rescue this framework?
